fastestimator/trace/metric/precision.py

```diff
@@ -72,7 +79,23 @@ def on_batch_end(self, data: Data) -> None:
 
     def on_epoch_end(self, data: Data) -> None:
         if self.binary_classification:
-            score = precision_score(self.y_true, self.y_pred, average='binary')
+            score = precision_score(self.y_true, self.y_pred, average='binary', **self.kwargs)
         else:
-            score = precision_score(self.y_true, self.y_pred, average=None)
+            score = precision_score(self.y_true, self.y_pred, average=None, **self.kwargs)
```

Comment:
what does it do if we just pass conflicting kwargs without sanitizing first?

Reply:
I think sanitizing first is the best practice.